### PR TITLE
Update lcov to 1.15

### PR DESCRIPTION
**Pull request by iarspider**

### CODE REVIEW

**Comment by iarspider** [comment]
please test

**Comment by iarspider** [comment]
please test

**Comment by iarspider** [comment]
please test

**Comment by iarspider** [comment]
please test

File: lcov.spec [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
 ## NOCOMPILER
-Source: http://heanet.dl.sourceforge.net/sourceforge/ltp/%{n}-%{realversion}.tar.gz
+Source: https://github.com/linux-test-project/%{n}/archive/refs/tags/v%{realversion}.tar.gz
 Patch0: lcov-merge-files-in-same-dir
 
 
 %prep
 %setup -n %{n}-%{realversion}
-%patch0 -p1
+%patch0 -p0
```
Comment (smuzaffar) [inline lcov.spec:9]: this should be `-p1`

Comment (iarspider) [reply lcov.spec:9]: No, it should be `-p0`, but I forgot to keep `bin` in the path.

**Comment by iarspider** [comment]
This update will pull in more PERL dependencies, which is undesired.

**Pull request closed without merging**